lib/Sema/DerivedConformanceCodingKey.cpp

```diff
@@ -214,7 +214,8 @@ deriveBodyCodingKey_enum_stringValue(AbstractFunctionDecl *strValDecl, void *) {
     for (auto *elt : elements) {
       auto *baseTE = TypeExpr::createImplicit(enumType, C);
       auto *pat = new (C) EnumElementPattern(baseTE, SourceLoc(), DeclNameLoc(),
-                                             DeclNameRef(), elt, nullptr);
+                                             DeclNameRef(), elt, nullptr,
+                                             /*DC*/ strValDecl);
```

Comment:
Nit: I think the preferred style for these kinds of named arguments is the following, which is detected by `clang-format` and removes the space after the comment.

Same below

```suggestion
                                             /*DC=*/strValDecl);
```

Reply:
Seems like clang-format is happy with either style, if the `=` isn't present, it inserts a space, otherwise it removes it. Both styles seem to be widely in use in the codebase:

```
swift on pattern-requests
❯ rg "/\*\w+\*/" lib | wc -l
    2331

swift on pattern-requests
❯ rg "/\*\w+=\*/" lib | wc -l
    2893
```

Personally I much prefer the variant without the `=` and with the space, it feels less claustrophobic. https://llvm.org/docs/CodingStandards.html#comment-formatting does give an example of a comment that uses the `=`, but it's unclear if it's saying that it's the preferred style.

I can change this if you want, but I'll do it as a follow-up.

Reply:
I prefer `/*<ID>=*/` because that's what we mostly use in the solver but I don't usually pay much attention to formatting there.

Reply:
> Seems like clang-format is happy with either style, if the = isn't present, it inserts a space, otherwise it removes it. 

FWIW I'm pretty sure the "happy with either style" is really just because it adds whitespace around `/**/` normally (as opposed to the special case when it has `=`). I'm surprised how many non-= there are, I've basically always seen = 🤔

Reply:
I think it varies by component, e.g:

| | With `=` | Without `=` |
|---|---|---|
| IDE | 170 | 41 |
| IRGen | 420 | 144 |

Reply:
I think it varies by component, e.g:

| | With `=` | Without `=` |
|---|---|---|
| IDE | 170 | 41 |
| IRGen | 420 | 144 |
| SILOptimizer | 122 | 260 |

Reply:
Hmm, OK. I didn’t know it was this mixed. It’s not a blocker for me either, just something I noticed. I’m happy to merge it as-is as well.